settings/dconf-defaults/settings

```diff
@@ -14,4 +14,13 @@
 # doesn’t care.
 # https://phabricator.endlessm.com/T20990
 [org/gnome/desktop/notifications/application/gnome-power-panel]
-show-in-lock-screen=false
\ No newline at end of file
+show-in-lock-screen=false
+
+# Preset the enabled-extensions gsetting to enable mandatory shell extensions on Endless OS.
+# https://help.gnome.org/admin/system-admin-guide/stable/extensions-lockdown.html.en
+# https://phabricator.endlessm.com/T28777
+[org/gnome/shell]
+enabled-extensions=['ubuntu-appindicators@ubuntu.com']
+development-tools=false
+disabled-extensions=['']
+disable-user-extensions=true
```

Comment:
This doesn't mean that it will disable  user-installed extensions **only**. This disables all extensions at once, without affecting the `enabled-extensions` / `disabled-extensions` lists.

Hence, we need to revert this key's value